src/helpers/DataRequirementHelpers.ts

```diff
@@ -36,6 +45,84 @@ export function flattenFilters(filter: AnyFilter): AnyFilter[] {
   }
 }
 
+/**
+ * Returns a
```

Comment:
This should be updated - maybe something like "Returns FHIR library containing data requirements, given a root library"?

Reply:
Hahaha oops- thanks!